cves/CVE-2007-6388.yml

```diff
@@ -78,7 +85,11 @@ unit_tested:
 
     For the "fix" answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
-  answer:
+  answer: |
+    There do not appear to be unit tests within the file. Not sure if they exist
+    apart from the file. I think it is likely that they were used, because of the
+    other XSS issues announced at the same time. I would imagine that one
+    was found incidentally and the others were discovered through testing.
```

Comment:
Was there any unit tests done after the fix? If not, what type of testing did they do after the fix?

Reply:
I didn't see any testing other than to make sure the fix was working.